TESTING.md

```diff
@@ -58,19 +58,30 @@ Run `mvn verify` from the project root to run all tests. Some Maven goals will n
 
 While the Java Cryptography Extensions are expected to be installed, it is
 possible to run a subset of the test suite by adding
-`-DexcludedGroups=unlimited-crypto`, e.g.:
+`-Dtestnames=Manta Client Error Tests,Manta Client Timeout Tests`, e.g
 ```
-mvn test -DexcludedGroups=unlimited-crypto
+mvn test -Dtestnames=Manta Client Error Tests,Manta Client Timeout Tests
```

Comment:
This example doesn't work. Did you test it?

Reply:
It seems [Test-NG Docs](https://testng.org/doc/documentation-main.html#running-testng) haven't been updated as to the fact that Test-Ng runner parameters for `-testname`, `-testnames` aren't supported anymore, should've verified the changes before checking them in, in my perspective it will be wise to **revert** back to using `-groups` as *test runner params* .